xml/bp_chap_getting_started_with_salt_formulas.xml

```diff
@@ -475,12 +481,44 @@ timezone:
       a user sees when hovering over a field and
       <literal>$placeholder</literal> displays a gray placeholder text
       in the field. <literal>$placeholder</literal> may only be used
-      with text fields like text, password, email or date. It doesnt
+      with text fields like text, password, email or date. It does not
       make sense to add a placeholder if you also use
       <literal>$default</literal> as this will hide the placeholder.
      </para>
     </listitem>
    </varlistentry>
+
+   <varlistentry>
+    <term>$minItems and $maxItems</term>
+    <listitem>
+     <para>
+      In an <literal>edit-group</literal>,
+     <literal>$minItems</literal> and <literal>$maxItems</literal> allow
+     you to specify the lowest and highest number the group can occur.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>$itemName</term>
+    <listitem>
+     <para>
+      In an <literal>edit-group</literal>, <literal>$itemName</literal>
+      allows to define a template for the name to used for the members
+      of the group.
+     </para>
+    </listitem>
+   </varlistentry>
+
+   <varlistentry>
+    <term>$prototype</term>
+    <listitem>
+     <para>
+      In an <literal>edit-group</literal>, <literal>$prototype</literal>
+      allows to define a template to used for the members of the group.
```

Comment:
In this case it's about the default values for the new group members, maybe i'd mention this explicitly:
```
...allows to define default (or pre-filled) values for newly added members in the group...
```